Source/WebCore/rendering/PathOperation.h

```diff
@@ -236,13 +231,7 @@ class RayPathOperation final : public PathOperation {
         Sides
     };
 
-    static Ref<RayPathOperation> create(float angle, Size size, bool isContaining)
-    {
-        return adoptRef(*new RayPathOperation(angle, size, isContaining));
-    }
-
-    WEBCORE_EXPORT static Ref<RayPathOperation> create(float angle, Size, bool isContaining, FloatRect&& containingBlockBoundingRect, FloatPoint&& position);
-
+    WEBCORE_EXPORT static Ref<RayPathOperation> create(float angle, Size size, bool isContaining);
```

Comment:
Let's remove the `size` name as the style checker says :)

Reply:
Fixed.